### PR TITLE
[ENG-30192] Edge sql api ref

### PR DESCRIPTION
**Pull request by gabriel-azion**
related issue: https://aziontech.atlassian.net/browse/ENG-30192

note: 

I will review the code samples and provide better ones. Focus on the text

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: Edge SQL API 
+description: The Edge Functions SQL API is an interface that allows access to the Azion Edge SQL. Through this interface, it's possible to read and write data in the storage and its buckets.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:3]:
```suggestion
description: The Edge Functions SQL API is an interface for access to Azion Edge SQL to read and write data in the storage and its buckets.
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+meta_tags: edge function, edge computing, storage, jamstack, apis
+namespace: documentation_runtime_edge_sql
+permalink: /documentation/devtools/runtime/api-reference/edge-sql/
+menu_namespace: runtimeMenu
+---
+
+The Edge SQL API provides an interface for interacting with an [EdgeSQL]() database from Edge Functions running on the Azion Edge Platform. This document outlines the key components of the API and provides code samples for each.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:10]:
```suggestion
The **Edge SQL API** provides an interface for interacting with an [EdgeSQL](/en/documentation/products/store/edge-sql/) database from Edge Functions running on the Azion Edge Platform. This document outlines the key components of the API and provides code samples for each.
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+## Database
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:12]:
```suggestion

---

## Database  
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+## Database  
+
+Creates a connection to an EdgeSQL Database for your application. Returns a Connection type object.
+
+### Methods
+
+Method    | Description | Signature | Parameters | Return
+---       |    :---:    |    :---:  |    :---:  |  :---:
+Open      | Opens a connection to the read replica of an EdgeSQL database | `static async open(name)` | `name: <string>` | `<Connection>`
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:20]:
```suggestion
Method  | Description | Signature | Parameters | Return |
--- | :---: | :---: | :---: |  :---: |
Open | Opens a connection to the read replica of an EdgeSQL database | `static async open(name)` | `name: <string>` | `<Connection>`
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+namespace: documentation_runtime_edge_sql
+permalink: /documentation/devtools/runtime/api-reference/edge-sql/
+menu_namespace: runtimeMenu
+---
+
+The Edge SQL API provides an interface for interacting with an [EdgeSQL]() database from Edge Functions running on the Azion Edge Platform. This document outlines the key components of the API and provides code samples for each.
+
+## Database  
+
+Creates a connection to an EdgeSQL Database for your application. Returns a Connection type object.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:14]:
```suggestion
Creates a connection to an EdgeSQL database for your application. Returns a **Connection** type object.
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+import { Database } from "azion:sql";
+
+const db = await Database.open('my_database');
+```
+
+---
+
+## Connection
+
+Represents the communication channel with a specific EdgeSQL Database. This object can be obtained by opening a database.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:34]:
```suggestion
Represents the communication channel with a specific EdgeSQL database. This object can be obtained by opening a database.
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+## Statement
+
+An abstraction of an SQL command, the Statement object allows for the execution of the represented command. It can be created from a `Connection.prepare` command.
+
+### Methods
+
+Method   | Description | Signature | Parameters  | Return
+---      |    ---    |    ---  |   ---     |  ---
+Execute  | Executes an SQL command | `async execute(sql, params)` | `sql: [<string>] '?'  positional parameter :<param_name> params: [<param>] Array of positional values Dictionary of values` | `-`
+Query    | Executes an SQL command | `async query(sql, params)` | `sql: [<string>] '?'  positional parameter :<param_name> params: [<param>] Array of positional values Dictionary of values` | `<Rows>`
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:81]: same description here?

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+
+const row = await result.next();
+
+console.log(row.columnName(0)); // prints the name of the first column
+console.log(row.columnType(0)); // prints the type of the first column
+console.log(row.getValue(0)); // prints the value of the first column
+console.log(row.getString(0)); // prints the value of the first column as a string
+```
+
+`*Note: <Value> could be a number, bigint, string, or boolean.`
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:180]:
```suggestion
> <Value> could be a number, bigint, string, or boolean.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:180]: use >  or the note component

File: src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+---         |    ---    |    ---  |    ---   |  ---
+`columnName` | Returns the column's name | `columnName(index)` |  `index: int32`  | `String`
+`columnType` | Returns the column's type | `ColumnType(index)` |  `index: int32`  | `int32`
+`getValue`   | Returns the attribute value | `getValue(index)` |  `index: int32`  | `Value`
+`getString`  | Returns the attribute value as string | `getString(index)` |  `index: int32`  | `String`
+
+## Code sample 
+
+The code sample shown below presents a way to interact with a database and retrieve data from a table. In this example, we're using a db called `mydatabase` and the table `users`.
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/edge-sql/edge-sql.mdx:98]:
```suggestion
The code sample shown below presents a way to interact with a database and retrieve data from a table. This example uses a db called `mydatabase` and the table `users`.
```
```

**Pull request merged**